README.rst

```diff
@@ -24,8 +24,19 @@ Getting Started
 
 Complete documentation is available at `zazu.readthedocs.io <http://zazu.readthedocs.io>`__.
 
-Zazu is a CLI development workflow management tool that combines
-elements of git flow with issue tracking and code review.
+Zazu is a CLI development workflow management tool that works with git, github, and JIRA.
+
+Features at a glance
+--------------------
+
+- Branch creation from new or existing tickets.
+- Issue creation to associate branch with issue.
+- Code review creation to associate review with issue.
+- Code style enforcement/fix on commit (via git hook).
+- Inserts branch name into every commit (via git hook).
+- Open code review and issue webpages from terminal.
+- `git-flow <https://nvie.com/posts/a-successful-git-branching-model/>`__ branching model.
```

Comment:
This is optional right ? user can specify the base branch.

Reply:
Yes, that's true. But so is the git flow tool...I can just say "configurable"